### PR TITLE
Fix missing logs

### LINKED ISSUE #30: Persist checkpoint file
Scalyr agent uses a checkpoint file to avoid log duplication. This should be highlighted/used in a Kubernetes cluster since the file should survive agent container restarts.

### PR DESCRIPTION
**Pull request by mohabusama**
- Adjust Scalyr config with ``copy_from_start`` option.
- Adjust rate limits config for journald monitor plugin.
- Update README with adjusted volume types and initContainer (to avoid log duplication).
- More debug statements.
- Patch pykube client session ``trust_env``. (eliminates the need for ``update-ca-certificates``).

Resolve #30 #28 
Possibly fixing #31

### CODE REVIEW

File: kube_log_watcher/agents/scalyr.py [hunk 1]
```diff
@@ -31,6 +31,15 @@ def __init__(self, cluster_id: str, load_template):
                 'Scalyr watcher agent initialization failed. {} config path does not exist.'.format(
                     self.config_path))
 
+        if not os.path.exists(self.dest_path):
```
Comment (lmineiro) [inline kube_log_watcher/agents/scalyr.py]: I expect us not to create it lazily due to concurrency...?

**Comment by lmineiro** [comment]
Although I can't, in all honesty, grasp parts of this, it LGTM

**Comment by lmineiro** [comment]
👍

File: kube_log_watcher/agents/scalyr.py (hunk 1 above)
Comment (mohabusama) [inline kube_log_watcher/agents/scalyr.py]: It is a kubernetes volume, so it must be there in order to function properly. This path is actually shared with Scalyr-agent.

**Comment by mohabusama** [comment]
👍

**Comment by chen-anders** [comment]
`initContainer` seems to crash loop with no good error output. We're on k8s 1.5.5 at the moment.

**Comment by hjacobs** [comment]
@chen-anders thanks for the feedback, can you maybe open a new issue for that?

**Pull request merged**